app/assets/stylesheets/common/_markus.scss

```diff
@@ -1003,11 +1003,12 @@ nav {
     display: none;
     left: -1px;
     margin: 0;
-    min-width: 16ch;
     padding: 0;
     position: absolute;
     text-align: left;
-    top: 1.95em;
+    top: 2.2em;
```

Comment:
I'm not sure why you changed this one?

Reply:
1.95em slightly overlays the display name box

Reply:
@therealfake it looks fine to me. Please revert.